code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -75,7 +75,7 @@
 	name = "xray laser gun"
 	desc = "A high-power laser gun capable of expelling concentrated xray blasts that pass through multiple soft targets and heavier materials"
 	icon_state = "xray"
-	item_state = "laser"
+	item_state = null
 	origin_tech = "combat=5;materials=3;magnets=2;syndicate=2"

```

Comment:
I would like to note here that xray lasers, which require combat 6, mat 5, bio 5, and power 4 to produce, give combat 5, mat 3, MAGNETS 2, and SYNDICATE 2.